src/components/Footer/Footer.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import {Container, Segment, Label, List} from 'semantic-ui-react';
+import {systemEnvColors} from '../../common/enums';
+
+const Footer = () => {
+  const lastVersion = process.env.REACT_APP_LAST_VERSION;
+
+  var syslevel = 'local';
```

Comment:
Can we make sure to include qa, too? It may be easiest to just include another environment variable for syslevel, but you could probably also figure it out based on the git vars.